controllers/cartProductController.ts

```diff
@@ -0,0 +1,25 @@
+import { Request, Response } from 'express';
+import {addProductToCart, getMyCart} from '../services/cartProductServices';
+import { sendResponse } from '../utils/util';
+
+export const addProductToCartController = async (req: Request, res: Response) => {
```

Comment:
the function itself is not a controller, the file or the list of function in the file called a controller. So the names here are better if they doesn't contain `Controller` at the end of them.